source/_patterns/01-molecules/content-header/content-header.twig

```diff
@@ -0,0 +1,40 @@
+{% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-header', 'wrapper']) }) %}
+
+<div {%- include 'atoms-attributes' %}>
+
+  {%- block body -%}
+
+    {%- with contentTitle only -%}
+
+      {%- block title -%}
+
+        {% set length = include('atoms-heading')|striptags|trim|length %}
```

Comment:
This is a little annoying, as it means rendering it twice.

Reply:
Can be a problem if the rendering is not deterministic e.g. generating an HTML id